### DISCUSSION TITLE
How to make Automatic start and stop

### DISCUSSION THREAD

**Discussion by pullasurya** [sorayuki/obs-multi-rtmp#117] [Q&A]
Hello guys,
I installed RTMP plugin and its great but I looking for how to start  and stop it automatically. i welcome your suggestions.
thank you

**Comment by chticer** [comment]
I would like something like this too. Maybe a command line argument or a checkbox in the multiple output settings that can automatically start when the start streaming button is clicked? Definitely will help no matter what implementation is made as long as one can be made.

**Comment by ibuntuos** [comment]
Me I also really would like to have this feature maybe together with a time schedule so that it doesn't need someone in person to do that. My current solution is really bumpy: With TinyTask I simulate someone who is clicking the Button and with windows scheduler I let this program run to the times I want to start/stop the stream. Would love if there would be a build-in-solution

**Comment by chticer** [comment]
Just noticed the 0.2.8 update that there's a checkbox that can start multiple streams automatically. It will be a few days before I can try it out but love that it is there now. Thank you @sorayuki for implementing this.

**Reply by JeffKnight** [reply]
This used to be an option in previous versions too. I don't know why the button disappeared.

**Reply by sorayuki** [reply]
> This used to be an option in previous versions too. I don't know why the button disappeared.

I'll check if I make changes based on a wrong commit.

**Reply by JeffKnight** [reply]
It works fine in the 0.2.8 release, but not the ones before it. I spent 2 hours troubleshooting it this morning... then updated to v28 and realized it was working there but my NDI plugin didn't work so had to revert back to v27. Did some trial and error with previous versions of the plugin until by chance I actually read the "Other Links" part of your post and tried that one and it suddenly worked!

**Reply by ComputerMonster** [reply]
Super late to the game, did this get resolved? I'm not sure I see a way to auto start/stop multiple RTMP streams in the plugin.

**Reply by Yagami-kun** [reply]
> Super late to the game, did this get resolved? I'm not sure I see a way to auto start/stop multiple RTMP streams in the plugin.

Thats works well here e.e

**Reply by ComputerMonster** [reply]
How? I don't see an option in the software, can you send a screenshot of the setting?

**Reply by MrCussMustard** [reply]
Sync-start with OBS and Sync-stop with OBS in the lower-right of the connection settings window causes the stream to start and stop when main obs stream begins and ends.
